### PR TITLE
Internalization has been added

### PR DESCRIPTION
**Pull request by lihorecoco**
- i18n custom direcitive has been created
- Proper static text has been changed with i18n attribute

### CODE REVIEW

**Comment by lihorecoco** [comment]
+1

File: config/paths/source-path.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 var sourceList = [
     "app/**/*.js",
+    "module/common/i18n/module.js",
```
Comment (tunix) [inline config/paths/source-path.js:3]: this file is already being copied by below line?

File: src/js/module/common/i18n/i18n.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+angular.module("gerritDashboard.i18n").directive("i18n",
+    ["i18n",
+        function (i18n) {
+            "use strict";
+
+            return {
+                restrict: "A",
```
Comment (tunix) [inline src/js/module/common/i18n/i18n.js:7]: what is this "A"?

File: src/js/module/common/i18n/i18nProvider.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+}]);
+
+angular.module("gerritDashboard").config(
+    ["i18nProvider", "$windowProvider",
+        function (i18n, $windowProvider) {
+            var $window = $windowProvider.$get();
+            var lang = $window.localStorage.getItem(i18n.getLanguageStorageKey) || "en";
+            i18n.setLanguage(lang);
+        }
+    ]);
```
Comment (tunix) [inline src/js/module/common/i18n/i18nProvider.js:71]: How are we going to change language preference? Will there be something in the UI?

File: src/js/module/common/i18n/translations.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        "count": "Count",
+        "average_merge_time": "Average Merge Time",
+        "average_first_review_time": "Average First Review Time",
+        "today": "Today",
+        "this_week": "This Week",
+        "this_month": "This Month",
+        "last_30_days": "Last 30 Days",
+        "last_6_months": "Last 6 Months"
+    }
+}]);
```
Comment (tunix) [inline src/js/module/common/i18n/translations.js]: please add the Turkish translation as well.

**Review by tunix — COMMENTED** [review]
thank you for doing this 👍 please make sure someone from the web team also reviews this.

File: config/paths/source-path.js (hunk 1 above)
Comment (lihorecoco) [reply config/paths/source-path.js:3]: i18n module has been created in there. It should  come before it's been used.

File: src/js/module/common/i18n/i18n.js (hunk 2 above)
Comment (lihorecoco) [reply src/js/module/common/i18n/i18n.js:7]: "A" stands for "Attribute" means this directive can be used as an attribute

File: src/js/module/common/i18n/i18nProvider.js (hunk 3 above)
Comment (lihorecoco) [reply src/js/module/common/i18n/i18nProvider.js:71]: There may be flag icon on the header to change lang

File: src/js/module/common/i18n/translations.js (hunk 4 above)
Comment (lihorecoco) [reply src/js/module/common/i18n/translations.js]: I will add soon

File: config/paths/source-path.js (hunk 1 above)
Comment (tunix) [reply config/paths/source-path.js:3]: 👍

File: src/js/module/common/i18n/i18n.js (hunk 2 above)
Comment (tunix) [reply src/js/module/common/i18n/i18n.js:7]: 👍

File: src/js/module/common/i18n/i18nProvider.js (hunk 3 above)
Comment (tunix) [reply src/js/module/common/i18n/i18nProvider.js:71]: Could you please add that as well? 😊

**Comment by mustafaersahin** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/monitise-mea/gerrit-dashboard-web/pull-request/13/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=monitise-mea/gerrit-dashboard-web)

**Pull request merged**